.npmignore

```diff
@@ -1 +0,0 @@
-src/
```

Comment:
This is so that a published NPM module doesn't publish the `src/` directory. We probably still want this. This will remove the tests as well, as they're in src.